@kiva/kv-components/vue/KvCarousel.vue

```diff
@@ -1,11 +1,12 @@
 <template>
 	<section
-		ref="KvCarousel"
+		ref="rootEl"
 		class="tw-overflow-hidden tw-w-full"
 		aria-label="carousel"
 	>
 		<!-- Carousel Content -->
 		<div
+			role="carousel"
```

Comment:
I don't think `carousel` is a recognized ARIA role, because I don't see it in this list https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles. I took a look at this W3C guide which doesn't appear to use roles at all for carousels https://www.w3.org/WAI/tutorials/carousels/structure/. 

Was this role added for the 'renders with a role of "carousel"' test? If so, I would recommend not implementing that test, as I don't think this carousel needs to have a role.

Reply:
Yeah, it was added for testing, but removed now.